### PR TITLE
Feature/4621 save data

### PR DESCRIPTION
**Pull request by alibombaye**
Story [AB#4621](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/4621)
Task [AB#7114](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7114)

### CODE REVIEW

File: app/pages/sections/supplier/supplier/controller.test.js [hunk 1]
```diff
@@ -39,4 +39,64 @@ describe('supplier controller', () => {
...
+      expect(putData.mock.calls.length).toEqual(1);
+      expect(putData).toHaveBeenCalledWith({
+        endpoint: `${orderApiUrl}/api/v1/orders/order-id/sections/supplier`,
+        body: formattedPutData,
+        accessToken: 'access_token',
+        logger,
+      });
+    });
+
+    it('should return succes: true if put is successful', async () => {
```
Comment (kat1906) [inline app/pages/sections/supplier/supplier/controller.test.js:93]: need tests for 400 or non-400

Comment (alibombaye) [reply app/pages/sections/supplier/supplier/controller.test.js:93]: I'm doing that as part of the next task. My bad I took bit of a shortcut and copy and pasted the implementation anyway.

Comment (kat1906) [reply app/pages/sections/supplier/supplier/controller.test.js:93]: tut tut very bad 😄

Comment (alibombaye) [reply app/pages/sections/supplier/supplier/controller.test.js:93]: I know I feel dirty 😞

**Pull request merged**